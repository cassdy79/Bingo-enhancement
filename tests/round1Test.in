1
a
b
1 Y 2
2 Y 5
3 B 3
4 U 1
5 U 5
0 Y 4
0 U 5
0 L 2
0 B 3
save round1Test
4