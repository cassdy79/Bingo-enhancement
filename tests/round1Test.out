a
b
3
0
0
F...............
LUUU
YBBU
RBLL
RRYL
BLLU
byrul
lbYru
ulByr
rulby
yrUlb
byrUl
Lbyru
ulbyr
rulby
yrulb
.
..
...
....
.....
.
..
...
YY..
YYY..
       
       

RRYBYRRLLBUURLYRBRUBLLBBBRRRBRUBRRUBBYLBLBLYULBYYRLUBYYURLUYULUYLYRR
23